File: trunk/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/EntryTypeNotExistException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class EntryTypeNotExistException extends Exception {

    /**
     * 
     */
    private static final long serialVersionUID = 1L;

    public EntryTypeNotExistException() {
        super("Entry type not exists.");
    }
}

File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/EntryTypeNotExistException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class EntryTypeNotExistException : Exception
    {
        public EntryTypeNotExistException()
            : base("Entry type not exists.")
        {
        }
    }
}
